sympy/geometry/ellipse.py

```diff
@@ -1268,6 +1268,45 @@ def vradius(self):
         return self.args[2]
 
 
+    def second_moment_of_area(self, point=None):
+        """Returns the second moment of area of a ellipse.
+
+        Parameters
+        ==========
+
+        point : Point, two-tuple of sympifiable objects, or None(default=None)
+            point is the point about which second moment of area is to be found.
+            If "point=None" it will be calculated about the axis passing through the
+            centroid of the ellipse.
+
+        Examples
+        ========
+        >>> from sympy import Point, Ellipse
+        >>> p1 = Point(0, 0)
+        >>> e1 = Ellipse(p1, 3, 1)
+        >>> e1.second_moment_of_area()
+        (3*pi/4, 27*pi/4)
+
+        References
+        ==========
+
+        https://en.wikipedia.org/wiki/List_of_second_moments_of_area
+
+        """
+
+        I_xx = (S.Pi*(self.hradius)*(self.vradius**3))/4
+        I_yy = (S.Pi*(self.hradius**3)*(self.vradius))/4
+
+        if(point == None):
```

Comment:
```python
if point is None:
```

parentheses are not needed, nor typical, and you should use `is` for `None` checking